moveit_core/collision_distance_field/CMakeLists.txt

```diff
@@ -14,28 +14,27 @@ add_library(${MOVEIT_LIB_NAME}
   src/collision_world_hybrid.cpp
 )
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
-target_link_libraries(${MOVEIT_LIB_NAME}
-  moveit_planning_scene moveit_distance_field
-  ${rclcpp_LIBRARIES} ${rmw_implementation_LIBRARIES} ${Boost_LIBRARIES})
-add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
 
-if(CATKIN_ENABLE_TESTING)
-  find_package(moveit_resources REQUIRED)
-  include_directories(${moveit_resources_INCLUDE_DIRS})
-
-  catkin_add_gtest(test_collision_distance_field test/test_collision_distance_field.cpp)
-  target_link_libraries(test_collision_distance_field  ${MOVEIT_LIB_NAME})
-endif()
+ament_target_dependencies(${MOVEIT_LIB_NAME}
+  moveit_planning_scene
+  moveit_distance_field
+  urdf
+  visualization_msgs
+  tf2_eigen
+)
 
-# install(TARGETS ${MOVEIT_LIB_NAME}
-#   ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-#   LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-#   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-# )
+install(TARGETS ${MOVEIT_LIB_NAME}
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)
 
-install(TARGETS talker
-  LIBRARY DESTINATION lib
-  RUNTIME DESTINATION bin
-)
+install(DIRECTORY include/ DESTINATION include)
 
-install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
+# if(BUILD_TESTING)
```

Comment:
I would remove this until we get some tests.

Reply:
On this case I prefer to add a TODO, more than removing it entirely